src/app/share-preview/components/share-preview/share-preview.component.ts

```diff
@@ -103,7 +104,7 @@ export class SharePreviewComponent implements OnInit, OnDestroy {
 	public hideBannerSubject: Subject<void> = new Subject<void>();
 	public hideBannerObservable = this.hideBannerSubject.asObservable();
 	public isUnlistedShare = false;
-	public ephemeralFolder: any = null;
+	public ephemeralFolder: FolderVO = null;
```

Comment:
I'm surprised this passes our tsc (maybe it infers that this is actually `FolderVO | null` -- but let's make that explicit.